prow/gerrit/adapter/adapter.go

```diff
@@ -261,6 +262,7 @@ func (c *Controller) Sync() {
 			changeChan <- Change{changeInfo: change, instance: instance, tracker: timeBeforeSent}
 		}
 		wg.Wait()
+		log.WithFields(logrus.Fields{"instance": instance, "duration(s)": time.Since(timeCompletedGettingChanges).Seconds()}).Info("Time taken for processing changes")
```

Comment:
@timwangmusic Can you try converting this to a Prometheus metric instead? I assume it would be straightforward. Plus it's OK if the exact parameters aren't correct in your first attempt because at least everything is explicit and Cole and review it before it's merged.

Reply:
Done.